open_academy/report/open_academy_session_report.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+    <record id="open_academy_session_report" model="ir.actions.report">
+        <field name="name">Session report</field>
+        <field name="model">open.academy.session</field>
+        <field name="report_type">qweb-pdf</field>
+        <field name="report_name">open_academy.report_session_template</field>
+        <field name="report_file">open_academy.report_session_template</field>
+        <field name="attachment_use" eval="True" />
+        <field name="binding_model_id" ref="model_open_academy_session" />
+        <field name="binding_type">report</field>
+    </record>
+
+    <template id="report_session_template">
+        <t t-call="web.html_container">
+            <t t-foreach="docs" t-as="doc">
+                <t t-call="web.external_layout">
+                    <div class="page">
+                        <h1 t-field="doc.name" class="mt0 float-left"></h1>
+                        <div class="row mt48 mb32">
+                            <div class="col-auto" name="div_start_date">
+                                <strong>Start date:</strong>
+                                <p t-field="doc.start_date" />
+                            </div>
+                            <div class="col-auto" name="div_end_date">
+                                <strong>End date:</strong>
+                                <p t-field="doc.end_date" />
+                            </div>
+                        </div>
+                        <br />
+                        <h2 class="mt0 float-left">Attendees</h2>
```

Comment:
Missing translation for this term (is not present in es.po file).

Reply:
![image](https://user-images.githubusercontent.com/108700860/184411447-aededed6-b072-4eb5-9531-47f333f0061c.png)
for some reazon not even generating again the es.po file the traduction apper as <h2>Attendees<h2/> but as you can see in the image above it translate it correctly